oak-search-elastic/src/main/java/org/apache/jackrabbit/oak/plugins/index/elastic/ElasticConnection.java

```diff
@@ -118,6 +119,9 @@ private Clients getClients() {
                         Header[] headers = new Header[]{new BasicHeader("Authorization", "ApiKey " + apiKeyAuth)};
                         builder.setDefaultHeaders(headers);
                     }
+                    builder.setRequestConfigCallback(
+                            requestConfigBuilder -> requestConfigBuilder.setConnectTimeout(ES_CLIENT_CONNECT_TIMEOUT).setSocketTimeout(ES_CLIENT_CONNECT_TIMEOUT));
```

Comment:
the default connectTimeout is 1s, while the socketTimeout is 30s. I think we should increase only the latter.